src/main/java/appeng/tile/misc/TileCellWorkbench.java

```diff
@@ -166,26 +166,27 @@ public void onChangeInventory( final IInventory inv, final int slot, final InvOp
 
 				if( cellHasConfig )
 				{
-					for( int x = 0; x < this.config.getSizeInventory(); x++ )
+					for( int x = 0; x < this.config.getSlots(); x++ )
 					{
-						this.config.setInventorySlotContents( x, configInventory.getStackInSlot( x ) );
+						this.config.setStackInSlot( x, configInventory.getStackInSlot( x ) );
 					}
 				}
 				else
 				{
-					for( int x = 0; x < this.config.getSizeInventory(); x++ )
+					for( int x = 0; x < this.config.getSlots(); x++ )
 					{
-						configInventory.setInventorySlotContents( x, this.config.getStackInSlot( x ) );
+						configInventory.setStackInSlot( x, this.config.getStackInSlot( x ) );
 					}
 
-					configInventory.markDirty();
+					// TODO: should not be needed anymore
+					// configInventory.markDirty();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)